native/src/main/java/io/ballerina/stdlib/sql/parameterprocessor/AbstractResultParameterProcessor.java

```diff
@@ -213,7 +225,141 @@ public abstract Object processCustomOutParameters(
             CallableStatement statement, int paramIndex, int sqlType) throws ApplicationError;
 
     public abstract Object processCustomTypeFromResultSet(ResultSet resultSet, int columnIndex,
-                                                           ColumnDefinition columnDefinition) throws ApplicationError;
+                                                           ColumnDefinition columnDefinition) throws ApplicationError,
+            SQLException;
+
+    public Object processArrayResult(ResultSet resultSet, int columnIndex, int sqlType, Type ballerinaType)
+            throws ApplicationError, SQLException {
+        return convertArray(resultSet.getArray(columnIndex), sqlType, ballerinaType);
```

Comment:
Let's change this to 2 lines as before. Else it's harder to read.

Reply:
done in https://github.com/ballerina-platform/module-ballerina-sql/pull/329/commits/9afd6cd860d444cb9e16b974ff74d0768c2cdb05